src/domains/messages-common/03_database.tf

```diff
@@ -0,0 +1,12 @@
+data "azurerm_cosmosdb_account" "cosmos_api" {
+  name                = format("%s-cosmos-api", local.product)
+  resource_group_name = format("%s-rg-internal", local.product)
+}
+
+resource "azurerm_key_vault_secret" "cosmos_api_master_key" {
```

Comment:
```suggestion
resource "azurerm_key_vault_secret" "cosmos_api_mongodb_connection_string" {
```

Reply:
Why mongo? we are pointing at our cosmosdb on IO :) On the other side we must ref master key because we are not working with a connection string indeed